src/shared/scrapers/BR/index.js

```diff
@@ -57,44 +57,111 @@ const scraper = {
     'São Paulo': ['iso2:BR-SP', 45919049, [-23.5505, -46.6333]],
     Tocantins: ['iso2:BR-TO', 1572866, [-10.1753, -48.2982]]
   },
-  async scraper() {
-    const response = [];
-    const ufs = this._ufs;
-    const $ = await fetch.headless(this, this.url, 'default');
-
-    $.root()
-      .find('.list-itens .teste')
-      .each(function() {
-        const uf = $(this)
-          .prev()
-          .text();
+  scraper: {
+    '0': async function scraper() {
+      const response = [];
+      const ufs = this._ufs;
+      const $ = await fetch.headless(this, this.url, 'default');
 
-        response.push({
-          state: ufs[uf][0],
-          cases: parseInt(
-            $(this)
-              .find('.lb-nome')
-              .eq(0)
-              .text(),
-            10
-          ),
-          deaths: parseInt(
+      $.root()
+        .find('.list-itens .teste')
+        .each(function() {
+          const uf = $(this)
+            .prev()
+            .text();
+
+          response.push({
+            state: ufs[uf][0],
+            cases: parseInt(
+              $(this)
+                .find('.lb-nome')
+                .eq(0)
+                .text(),
+              10
+            ),
+            deaths: parseInt(
+              $(this)
+                .find('.lb-nome')
+                .eq(1)
+                .text(),
+              10
+            ),
+            population: ufs[uf][1],
+            coordinates: [ufs[uf][2][1], ufs[uf][2][0]],
+            aggregate: 'state'
+          });
+        });
+
+      response.push(transform.sumData(response, { aggregate: 'state' }));
+
+      return response;
+    },
+
+    // TODO: things actually started busting before this point .. adjust this date as we figure out where it actually works.
```

Comment:
Yeah def run a timeseries and see the last date that worked:

```
yarn timeseries -d 2020-4-1
```

Reply:
Running for that location: `yarn timeseries --location BR -d 2020-4-1`, thx for tip.

Reply:
Hm, I just ran that ... and there was no failure.  So either BR just started failing today ... or something else happened, and it was never broken!  At any rate, the current scraper code will provide more failure data than before, should something go wrong.